ontology/mp.md

```diff
@@ -43,6 +43,12 @@ browsers:
   - label: Monarch
     title: Monarch Phenotype Page
     url: http://monarchinitiative.org/phenotype/MP:0000001
+usages:
+  - user: http://www.informatics.jax.org/vocab/mp_ontology
+    description: MGI uses MP for phenotype annotation
+    examples:
+      - url: http://www.informatics.jax.org/vocab/mp_ontology/MP:0005375
+        description: Adipose phenotype annotations
```

Comment:
@sbello provide better description of what kind of information one can access at that URL related to the MP term?